src/components/ui/ClusterEventsToolbar.tsx

```diff
@@ -121,7 +121,7 @@ const ClusterEventsToolbar: React.FC<ClustersListToolbarProps> = ({
     });
 
     let nextFilters = setNextSelectedValues(type, isChecked, value, filters);
-    nextFilters = setNextSelectAllValue(cluster.hosts?.length ?? 0, nextFilters);
+    nextFilters = setNextSelectAllValue(cluster.hosts?.length || 0, nextFilters);
```

Comment:
Our development build does not transpile this operator and OCM is not ready for it.

Reply:
+1. 
bumped into that too which working against uhc-portal